src/new/sendTransaction.js

```diff
@@ -36,8 +36,23 @@ import { OutputType } from '../wallet/types';
  * 'unexpected-error': if an unexpected error happens;
  **/
 class SendTransaction extends EventEmitter {
-  /*
-   * data {Object} Prepared tx data
+  /**
+   *
+   * @param {Transaction} [transaction] Full tx data
+   * @param {{
+   *   txId: string,
+   *   index: number
+   * }[]} inputs
+   * @param {{
+   *  type: string,
+   *  token: string,
+   *  data?: Buffer,
+   *  address: string,
+   *  value: number,
+   *  timelock: number}[]} outputs
+   * @param {string} [changeAddress]
+   * @param {string} pin
+   * @param {Network} network
```

Comment:
Implemented your suggestion in [21c8166b](https://github.com/HathorNetwork/hathor-wallet-lib/pull/406/commits/21c8166b9a41340542d4015f4e7543485314a29a) with just one change:

I left the `options` parameter without description and without a default value. This is because some _jsdocs_ interpreters get confused with the parameter initialization and cannot add the children properties later.